pkgs/development/python-modules/nassl/default.nix

```diff
@@ -36,7 +37,7 @@ let
     "enable-mdc2"
     "-fPIC"
   ];
-  opensslStatic = (openssl.override nasslOpensslArgs).overrideAttrs (
+  opensslStatic = (openssl_1_1.override nasslOpensslArgs).overrideAttrs (
     oldAttrs: rec {
       name = "openssl-${version}";
```

Comment:
```suggestion
      pname = "openssl";
```

Reply:
Nope, it needs `name` to fill out the download url.

Reply:
The openssl package in nixpkgs already uses pname + version in the download url. Maybe we should update it or not. I am not sure. Marking as unresolved so that it is clear where I wrote a comment.

Reply:
`overrideAttrs` needs to override name for the changes to be reflected. `pname`+`version` will be concatentated under `name` when invoked with `stdenv.mkDerivation`.

Reply:
This is done at multiple places wrong. I think this should change internally because it is very unintuitive.